src/apprt/gtk-ng/class/tab.zig

```diff
@@ -92,6 +92,21 @@ pub const Tab = extern struct {
     };
 
     pub const signals = struct {
+        /// The bell was rung by one of the child surfaces.
+        ///
+        /// The tab doesn't handle any functionality so it is up to the
+        /// embedding widget to react to this.
+        pub const bell = struct {
```

Comment:
We don't need to do this. See how we attach signals to Surface from Window.

Reply:
We should be able to bind to the "active surface". If I didn't hook that up yet then let me work on that. That'll come with splits but I thought I wired that property though already.

Reply:
I don't understand. I'll take a look when I context load on this more.